bosh-stemcell/lib/bosh/stemcell/builder_options.rb

```diff
@@ -63,6 +63,7 @@ def environment_variables
         'UBUNTU_ISO' => environment['UBUNTU_ISO'],
         'UBUNTU_MIRROR' => environment['UBUNTU_MIRROR'],
         'UBUNTU_ADVANTAGE_TOKEN' => environment['UBUNTU_ADVANTAGE_TOKEN'],
+        'IAAS_KERNEL' => environment['IAAS_KERNEL'],
```

Comment:
nit: should this be `UBUNTU_IAAS_KERNEL` given that this is Ubuntu specific?

Reply:
would be nice to check that this var can only be set to `aws` currently.

Reply:
renamed in https://github.com/cloudfoundry/bosh-linux-stemcell-builder/pull/309/commits/fed91af0a965655cfb6efa15085874f34de66863 and added a check for the value in https://github.com/cloudfoundry/bosh-linux-stemcell-builder/pull/309/commits/21485472676b68adf8bb711443abff0c92824d1c